PENS_v2.0.0_Draft.md

```diff
@@ -0,0 +1,1141 @@
+11 October 2016
+PENS v2.0.0 _**DRAFT**_
+
+DOCUMENT No. CMI010
+
+# Guidelines for Package Exchange Notification Services
+
+    VERSION 2.0.0
+	 _Proposed_ RELEASE DATE 2017-March-15  
+    Based on Revision 1.0a released 2006-March-15
+
+FORMATTING NOTES:
+
+This document is formatted using
+[Markdown](http://daringfireball.net/projects/markdown/). The document relies
+on features of GitHub Flavored Markdown ("GFM") as described at:
+<https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown>.
+For the most accurate rendering and to avoid potential misinterpretations due
+to formatting discrepancies, please view the documentation on GitHub or use a
+rendering engine supporting GFM.
+
+THIS DOCUMENT IS CONTROLLED BY:
+
+    cmi5 Workgroup, as successor to AICC CMI Subcommittee
+
+    ALL REVISIONS SHALL BE APPROVED BY THE ABOVE ORGANIZATION PRIOR TO RELEASE.
+    
+POINT OF CONTACT:
+
+Tom King  
+cmi5 PENS Workgroup Lead  
+<tking@onpointlearning.com>
+ 
+FILED UNDER: <https://github.com/AICC/PENS_Spec_Current>
+
+---
+
+### Specification for Learning Technology—  Package Exchange Notification Services (PENS)
+
+#### Abstract:
+
+This specification describes a protocol to support a notification service to
+announce the location of content package(s) that are available for transport.
+The intent is to automate the notification, transfer and delivery confirmation
+of content packages between tools or systems that generate content and systems
+that manage, publish or deliver content. The scope of the specification is
+specifically constrained to the notification request, package transfer and
+related responses. Specifically outside the scope of this specification are
+mechanisms for physical deployment of content packages, content management,
+version control, publication or revocation of content.
+
+#### Keywords:
+CBT, CMI, cmi5, CMS, cmi5, content management, content package, e-learning,
+LCMS, LMS, notification service, package exchange, PENS, SCORM, xAPI.
+
+#### Revision Notes:
+The PEN 2.0.0 revision focuses on four goals:
+
+1. **Package format updates**: The addition of contemporary package formats
+such as xAPI and cmi5.
```

Comment:
It's a white paper, but one that has seen a decent bit of adoption because it was the only thinking on xAPI launch out there for a long time. It may be that the only LMSs supporting it use our software, but I wouldn't be sure of that because it's easy enough to implement and some major authoring tools use it.

That said, I don't see why we would include it, I'm just saying that if we were to include anything for "xAPI without cmi5" launch (which we probably shouldn't), that white-paper should be it, because there just isn't currently any reasonable alternative to it. (I recall ADL was working on something else for launch, but it didn't seem to have a package format associated with it).

cmi5 can work rather well as a minimalistic packaging standard -- it doesn't require much of the content (assuming the content doesn't send any optional `cmi5` statements) that you wouldn't want in any minimalistic package and launch spec anyway. So it seems really reasonable to say that if you want to use xAPI with PENS, just use cmi5.